api/v1/candidates.coffee

```diff
@@ -1 +1,47 @@
+ObjectID = require('mongodb').ObjectID
+
 module.exports = (api, db) ->
+  api = api || {}
+  api.v1 = api.v1 || {}
+
+  api.v1.getCandidate = (req, res) ->
+    db.collection("candidates").findOne( {_id: new ObjectID(req.params.id)}, (err, results) ->
+      if err
+        res.jsonp
+          error: err
+        , 500
+
+        return
+
+      res.jsonp(results)
+    )
+
+  api.v1.getCandidates = (req, res) ->
+    api.v1.search(req, res)
```

Comment:
Oh, good catch. Maybe we should just get rid of the separate URL for search and have /api/v1/candidates accept the firstName/lastName parameters. Thoughts?